.github/workflows/test-notebook.yml

```diff
@@ -0,0 +1,94 @@
+name: notebook test
```

Comment:
This could be integrated into `whylogs-ci.yml` as extra steps in the `pithon-ci` job. Not sure which approach is best. Thoughts?

Reply:
I think this is better as standalone since it takes longer to run. I don't think we need to run it over the test matrix of python versions for example.